template/styles/sass/partials/_layout.scss

```diff
@@ -6,14 +6,13 @@
 }
 
 .body-content {
-    display: flex;
     position: relative;
-    overflow: hidden;
```

Comment:
Is there a reason overflow: hidden was removed?

Reply:
Yes, after removing the flex layout of the body content, the theming buttons started to behave strange on every vertical scroll the container flickered (only in IE): 
![flick](https://user-images.githubusercontent.com/25771040/79212515-6d08d400-7e50-11ea-94e1-ee7c3ed00169.gif)
[Here](https://stackoverflow.com/questions/20995478/fixed-positioned-element-flicker-in-ie-only-how-to-solve) I found that the overflow of one of the parent elements may cause it, so I removed it, since it also does not have any impact in other browsers